hypha/apply/users/templates/wagtailusers/users/results.html

```diff
@@ -3,10 +3,10 @@
     <div class="users-list__results">
         {% if users %}
             <h2 role="alert">
-            {% blocktrans trimmed count count=user_count %}
-                There is {{ user_count }} user
+            {% blocktrans trimmed count counter=users|length %}
```

Comment:
I think it will show the number of users present on the page not the total number of users.

Reply:
You are right the fix breaks on normal user list, thanks for catching that.